pkg/systemd/overview-cards/lastLogin.tsx

```diff
@@ -65,7 +72,7 @@ const LastLogin = () => {
         const bridge = cockpit.dbus(null, { bus: "internal" });
         try {
             const [reply] = await bridge.call("/LoginMessages", "cockpit.LoginMessages", "Get", []);
-            const obj = JSON.parse(reply);
+            const obj = JSON.parse(reply as string[0]);
```

Comment:
What is `as string[0]`?

Reply:
Ah, that's bogus, leftover from  some earlier variant. Thanks for spotting! Fixed to `as string`. The return value of `.call()` is currently `unknown[]`, I don't think we can restrict it much further. So we have to make explicit casts on method return values based on what we know about their signature.